providers/mock/mock.go

```diff
@@ -59,17 +59,18 @@ func NewMockProvider(providerConfig, nodeName, operatingSystem string, internalI
 }
 
 // loadConfig loads the given json configuration files.
-func loadConfig(providerConfig string) (config MockConfig, err error) {
-	if providerConfig != "" {
-		data, err := ioutil.ReadFile(providerConfig)
-		if err != nil {
-			return config, err
-		}
-		err = json.Unmarshal(data, &config)
-		if err != nil {
-			return config, err
-		}
+
+func loadConfig(providerConfig, nodeName string) (config MockConfig, err error) {
+	data, err := ioutil.ReadFile(providerConfig)
+	if err != nil {
+		return config, err
+	}
+	configMap := map[string]MockConfig{}
```

Comment:
it's not backward compatible

Reply:
@robbiezhang Hi, the config feature for mock provider is added by me just few days ago. I don't think others is already using it. So, should we still want to make it backward compatible?

Reply:
Sounds fair.